src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
@mikewallace1979 yes, because only these two fields are required for replication process. If the remote peer is a CouchDB database, it will provide both with no questions. If the remote is third party HTTP endpoint which claims to support replication protocol, this check will force it provides required for start information.